helpers/isChatToYourself.js

```diff
@@ -0,0 +1,29 @@
+const {User} = require('../databases/models');
+const UsersFunction = require('../databases/functions/users');
+
+module.exports = {
+  isChatToYourself: async (my_user_id, member) => {
+    if (my_user_id === member) return {success: false, message: 'Cannot chat to yourself'};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The comparison `my_user_id === member` assumes that both variables are of the same type. If there's a chance they could be different types (e.g., one is a string and the other is a number), consider using a loose equality check (`==`) or ensuring both variables are coerced to the same type before comparison.
```diff
-    if (my_user_id === member) return {success: false, message: 'Cannot chat to yourself'};
+    if (my_user_id == member) return {success: false, message: 'Cannot chat to yourself'};
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->